service_mesh/v2x/ossm-troubleshooting-istio.adoc

```diff
@@ -0,0 +1,74 @@
+
+// The following include statements pull in the module files that comprise the assembly.
+
+include::modules/ossm-understanding-versions.adoc[leveloffset=+1]
+
+== Troubleshooting Operator installation
+
+In addition to the information in this section, be sure to review the following topics:
+
+* xref:../../operators/understanding/olm-what-operators-are.adoc[What are Operators?].
+
+* xref:../../operators/understanding/olm/olm-understanding-olm.adoc[Operator Lifecycle Management concepts].
+
+* xref:../../support/troubleshooting/troubleshooting-operator-issues.adoc[OpenShift Operator troubleshooting section].
+
+* xref:../../support/troubleshooting/troubleshooting-installations.adoc[OpenShift installation troubleshooting section]
+
+include::modules/ossm-validating-operators.adoc[leveloffset=+2]
+
+include::modules/ossm-troubleshooting-operators.adoc[leveloffset=+2]
+
+== Troubleshooting the control plane
+
+The Istio control plane is composed of *Istiod* which consolidates several previous control plane components (Citadel, Galley, Pilot) into a single binary.   Deploying the `ServiceMeshControlPlane` also creates the other components that make up {ProductName} as described in the xref:../../service_mesh/v2x/ossm-architecture.adoc[architecture] topic.
+
+include::modules/ossm-validating-smcp.adoc[leveloffset=+2]
+
+include::modules/ossm-accessing-kiali.adoc[leveloffset=+3]
+
+include::modules/ossm-accessing-jaeger.adoc[leveloffset=+3]
+
+include::modules/ossm-troubleshooting-smcp.adoc[leveloffset=+2]
+
+== Troubleshooting the data plane
+
+The *data plane* is a set of intelligent proxies that intercept and control all inbound and outbound network communication between microservices in the service mesh.
+
+{ProductName} relies on a proxy sidecar within the application’s pod to provide Service Mesh capabilities to the application.
+
+include::modules/ossm-troubleshooting-injection.adoc[leveloffset=+2]
+
+For more information about sidecar injection, see xref:../../prepare-to-deploy-applications-ossm.adoc#ossm-automatic-sidecar-injection_deploying-applications-ossm[Enabling automatic injection]
+
+include::modules/ossm-troubleshooting-proxy.adoc[leveloffset=2]
+
+For more information about troubleshooting pod issues, see the xref:../../support/troubleshooting/investigating-pod-issues.adoc[OpenShift pod troubleshooting section]
+
+include::modules/support.adoc[leveloffset=+1]
+
+include::modules/support-knowledgebase-about.adoc[leveloffset=+2]
+
+include::modules/support-knowledgebase-search.adoc[leveloffset=+2]
+
+include::modules/about-must-gather.adoc[leveloffset=+2]
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+* The {product-title} CLI (`oc`) installed.
+
+include::modules/ossm-about-collecting-ossm-data.adoc[leveloffset=+2]
+
+For prompt support, supply diagnostic information for both {product-title} and {ProductName}.
+
+include::modules/support-submitting-a-case.adoc[leveloffset=+2]
```

Comment:
This seems like an odd place to have prereqs - maybe make this a == heading instead if the prereqs apply for a few modules?

Reply:
FYI: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies